### ISSUE TITLE
Vehicle sounds

### ISSUE THREAD

**Issue by Valkata4235421** [LividDS/lds-lorevehicles#1]
I followed the readme but no sound for the cars using the sounds in the txadmin they are started but when i spawn the car no sound

**Comment by the-game-stoner** [comment]
Hey idk if it would help you at all but if you use my fork of this I fixed all the audio issues. Just waiting for him to accept my pr

**Comment by itzboterberg** [comment]
can you upload it on your page?

**Comment by the-game-stoner** [comment]
> can you upload it on your page?

Yeah I've been waiting to see if they would accept my p.r I'll see what I can do after I get out of work

**Comment by itzboterberg** [comment]
thank you that will be greate!

**Comment by the-game-stoner** [comment]
> thank you that will be greate!

yeah man any time this link should be my branch with the fixes applied. 

https://github.com/the-game-stoner/lds-lorevehicles/tree/fix-audio

**Comment by LividDS** [comment]
@the-game-stoner 's merge has been accepted. Feel free to test it and let me know if it works!

PS: Life's been busy, so not much time to work on this..

**Comment by fatdesign** [comment]
How can I change the trunk size for each car??
